### PR TITLE
Fix #255: Split up `_boilerPlate()` in `transport.py`

### LINKED ISSUE #255: Split up `_boilerPlate()` in `transport.py`
`_boilerPlate()` currently does a few different things like fetching particle data, calculating thermal velocity, tests/checks. Each piece of functionality should be split into its own function.

See #191

### PR DESCRIPTION
**Pull request by apooravc**
Fixes #255 
Split _boilerPlate() into 5 functions, each performing only one task:

- Checking input
1) checkTemperature(T)
2) checkParticleSystem(particles)

- Obtaining parameters (masses, charges, reduced mass, thermal velocity of system of particles)
3) getParticleSystemInfo(particles)
4) getReducedMass(masses)
5) getVelocity(T, reduced_mass, V)

Removed some pep8 errors as well. @StanczakDominik Please have a look.

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Maybe we could prefix these function names with `_`, to make explicit that they're not expected to be used out of transport.py. I'm sorry, I'm not able to look at the code right now (on holidays and I think my girlfriend is going to kill me if I start programming).

File: plasmapy/physics/transport.py [hunk 1]
```diff
@@ -251,13 +260,21 @@ def _boilerPlate(T, particles, V):
         except Exception:
             raise ValueError("Unable to find charge of particle: "
                              f"{particles[i]}.")
+    return (masses, charges)
+
+
+def getReducedMass(masses):
     # obtaining reduced mass of 2 particle collision system
     reduced_mass = masses[0] * masses[1] / (masses[0] + masses[1])
+    return reduced_mass
```
Comment (StanczakDominik) [inline plasmapy/physics/transport.py]: I'm pretty sure we could avoid having to define this on our own here by using #262 or some oneliner with [the `Particle` class](http://plasmapy.readthedocs.io/en/latest/atomic/index.html).

File: plasmapy/physics/transport.py [hunk 2]
```diff
@@ -219,25 +219,34 @@ def Coulomb_logarithm(T,
...
+def checkParticleSystem(particles):
+    # checking particles input
     if not isinstance(particles, (list, tuple)) or len(particles) != 2:
         raise ValueError("Particles input must be a "
                          "list or tuple containing representations of two  "
                          f"charged particles. Got {particles} instead.")
+    return particles
 
+
+def getParticleSystemInfo(particles):
```
Comment (StanczakDominik) [inline plasmapy/physics/transport.py]: I think the two functions above could be merged. They're not doing anything special enough on their own to warrant being separate.

**Comment by apooravc** [comment]
@StanczakDominik Not at all. Please go ahead.

**Comment by lemmatum** [comment]
I think the segfault is coming from updated astropy. Probably need to rebase.

**Comment by lemmatum** [comment]
*poke*, how is this PR coming along?

**Comment by StanczakDominik** [comment]
Hey, I'm really sorry, but well... I kept working on other changes and now we don't even have a `transport.py` file I could merge these changes into. I'm sorry @apooravc , but I'll have to close this PR and make another one to fix this :(

**Pull request closed without merging**